.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -11,9 +11,16 @@ assignees: ''
 
 ### Which Crates to Increment
 
-To check if any of the top-level crates need version increments:
+Look for the [draft `zebrad` changelog](https://github.com/ZcashFoundation/zebra/releases) for the automatic version bump.
+This version is based on [the labels on the PRs in the release](https://github.com/ZcashFoundation/zebra/blob/main/.github/release-drafter.yml).
+
+Check that the automatic `zebrad` version increment is correct:
+1. If we're releasing a mainnet network upgrade, increment the `major` version of all Zebra crates
+2. Check for features, major changes, deprecations, and removals. If this release has any, it is a `minor` release
```

Comment:
> Another issue with this change is that this document already describes when to increment the `major` version in a different way in section **How to Increment Versions**.

Thanks, I removed the detailed information, because it seems like we can automate this a bit more, and save ourselves some time.